docs/integrations/skopt.rst

```diff
@@ -51,101 +106,90 @@ Everything can be inspected live.
     results = skopt.forest_minimize(objective, space, callback=[neptune_callback],
```

Comment:
I'd move the `callback` argument till te end and perhaps have it all "vertical" (one param per line or something like that) -> it will make it clear what changes.